RAILS_VERSION

```diff
@@ -1 +1 @@
-6.0.2
\ No newline at end of file
+6.0.3
```

Comment:
this solves the many warnings that are coming from rails dependencies https://weblog.rubyonrails.org/2020/5/6/Rails-6-0-3-has-been-released/